### PR TITLE
EngDebt: Proper module name, package names etc 

### PR DESCRIPTION
**Pull request by dhirenmathur**
Implements #25 
* Commit changes the following: 
Group id:  `com.example` -> `me`
Artefact id:  `rest-service` -> `simplq`
Imports and file structure: `com.example.rest-service` ==> `me.simplQ`

Tested with postman after making changes, all APIs behaving as expected. Can provide screenshots if necessary. 

Regarding deployment scripts: 
Please advice on `manifest.yaml` and `settings.gradle`, these files i believe still reference the old names.

### CODE REVIEW

**Comment by daltonfury42** [comment]
Hey. I'll be careful during deployment, and send follow up PRs where required.

1) Let's delete both `manifest.yaml` and `settings.gradle`. I think they are dead files. 

2) Let's make the group id `me.simplq` and artifact `simplq-backed`

3) In the folder structure, should we be using `simplq` instead of `simplQ`?

4) Also while we are are at it, let's rename `complete` folder to `simplq/java`. 

5) `Scripts/runJar.sh` can then move into `simplq/bash/run-jar.sh`

6) We would have to update `.appspec.yml`, `.buildspec.yml` and `.travis.yml`.
(The buildspec file is not required anymore as we are moving away from codebuild, but let's maintain it forsome more time)

I'm taking this as a reference for the folder structure: https://github.com/tensorflow/tensorflow/tree/master/tensorflow/java/src/main

**Comment by dhirenmathur** [comment]
Got it, I think it's better if I split this up into 2-3 smaller PRs to manage, otherwise it can become a mess. What do you say? 

1)Pom changes
2)File Structure
3)Moving scripts and updating yml file

**Comment by dhirenmathur** [comment]
>Also while we are are at it, let's rename complete folder to simplq/java.

I think renaming it to `simplq` or `simplq-server` is adequate, the tensorflow repo adds java as there are implementations of tensorflow in various other languages in that directory

**Comment by daltonfury42** [comment]
The tensorflow repo is organised that way because google follows a monorepo approach. Let's then just rename `complete/` to `simplq/` and the script folder can be named `scripts/run-jar.sh`.

I'd have preferred breaking to multiple PRs, but in this case, I'd want to merge it all at one go and fix any build issues. Since it's mostly IntelliJ refactoring and renaming, we can do in one PR.

**Comment by dhirenmathur** [comment]
Updated with the changes and verified functionality locally. See commit message for details of changes.

File: .travis.yml [hunk 1]
```diff
@@ -44,4 +44,3 @@ addons:
     organization: "simplq"
     token:
       secure: "BGWBr+ilSphwkx95mrG1GKXUtwg5q6ZYbYREQDEd3RtQsd+xH7EDblE8iuwJll8n7sWA7d8hV/nZoq28iM8mHna2Zn51iBQfKaK+vMmvAqYiPCls7H90uFCVZe2A3T4uo5vtJgpCSunSqigKql7Zs7TTeHDq9GI8QFS2Y46t7bnQMCymh6dVqqkc2KXO+AWaKcR0oxxO3rhCCtBb9FXSFZRgZF647HHm4tU1Uh2YIubkl2HOGdbKU3KjDy+PRF9QfpbnSifRjftjoDQQtDJRi+8Fd2qRq2Yf1qHcvo+Z04eyBlIs61xjNpeZvcJxFdVcH/aTHquJ0Bv6igQqbS51H4hwiG7CmMmmo3hc6ZRrvM4NcHYMmaVh0a7E+dtum63MnB5b1Qj3J/puNw+Cp8Lga02BR7qKyOq7FgB/c/qtCrtiFiQ8JLGQOZgLxJjgUiYtBzIDapYgng2mmIBE6sI4M29D0ROMVdrWeDQoC6yj7Yp0wvLhawtEh78XxZjBQ+yPzXSqVwQ+RfhSHF7KzOvrfoH/ISpR1aSQMgm4hRWByXsLuOi3nTe/CD0tI4BR7vE9Te9uYL8xZiAPau8q6cra5kJsdh2cweBSx/tDXHxQeQMWXWG5tYXPg9CSIpRBPG6bMaTPmfEKN1JN/ghzUuAioHHwKJeOu34hYsMg+b+SUSc="
-
```
Comment (dhirenmathur) [inline .travis.yml:47]:
```suggestion

```

**Review by daltonfury42 — APPROVED** [review]
Thanks for this.

**Pull request merged**